source/deprecations/v1.x.html.md

```diff
@@ -1023,6 +1023,38 @@ proxying behavior is being used.
 
 Added in [PR #11476](https://github.com/emberjs/ember.js/pull/11476).
 
+#### SortableMixin
+
+Along with `Ember.ArrayController`, `Ember.SortableMixin` will be removed in Ember 2.0.
+
+You can migrate away from using `Ember.SortableMixin` by using `Ember.computed.sort`. Using this example:
+
+```
+import Ember from 'ember';
+
+export default Ember.ArrayController.extend(Ember.SortableMixin, {
+  model: songs,
```

Comment:
I was just following the other examples for consistency ... I think they were assuming that `songs` would be populated via the router. I can break with the pattern here though.

Would you like to see a something thinking this:
```
  init: function() {
    this.super();
    this.set('songs', [ ... ]);
  }
```

Or would you rather me just emit the `model` definition all together and provide a footnote explaining that `model` is set by the route?